### ISSUE TITLE
Move trigger icon on window.resize

### ISSUE THREAD

**Issue by Globulopolis** [tsi/inlineDisqussions#12]
When we resize the browser window the plugin icon still on the old position, as before resize.

**Comment by tsi** [comment]
Yes, I never took care of browser resizing and re-positioning. we, developers, are the only ones to do that :)
I don't mind leaving this open if anyone would like to take this on.
